src/components/Toolbelt/Toolbelt.js

```diff
@@ -1,22 +1,25 @@
 import React from 'react'
 import Button from '@material-ui/core/Button'
 import Tooltip from '@material-ui/core/Tooltip'
-import { func, string } from 'prop-types'
+import PropTypes from 'prop-types'
```

Comment:
this is the standard i am used to and it's a bit easier to maintain, but i am open to whatever style is preferred in the codebase

Reply:
I personally prefer the destructured format, but not enough to impose it on others. I'm okay with a manageable mix of code styles in the codebase, so this is fine with me!